documentation-site/static/examples/tag/variants.js

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import {Tag} from 'baseui/tag';
+
+export default () => (
+  <>
+    <React.Fragment>
```

Comment:
do we need the ` <React.Fragment>` wrappers here?

Reply:
```suggestion
```

Reply:
These are only needed to maintain a row of `Tag` items being displayed. A simple `div` would have also sufficed.